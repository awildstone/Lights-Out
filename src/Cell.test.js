import React from "react";
import { render } from "@testing-library/react";
import Cell from "./Cell";

it("renders without crashing", () => {
    render(<Cell />);
});

it("matches snapshot when lit", () => {
    const { asFragment } = render(<Cell isLit={true} />);
    expect(asFragment()).toMatchSnapshot();
});

it("matches snapshot when not lit", () => {
    const { asFragment } = render(<Cell />);
    expect(asFragment()).toMatchSnapshot();
});