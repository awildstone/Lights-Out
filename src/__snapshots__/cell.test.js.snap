// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot when lit 1`] = `
<DocumentFragment>
  <td
    class="Cell Cell-lit"
    data-testid="light"
  />
</DocumentFragment>
`;

exports[`matches snapshot when not lit 1`] = `
<DocumentFragment>
  <td
    class="Cell "
    data-testid="light"
  />
</DocumentFragment>
`;
